engine/apps/api_for_grafana_incident/views.py

```diff
@@ -1,12 +1,18 @@
-from rest_framework.viewsets import ReadOnlyModelViewSet
+from rest_framework import mixins, viewsets
 
 from apps.alerts.models import AlertGroup
 from apps.auth_token.auth import GrafanaIncidentStaticKeyAuth
 
 from .serializers import AlertGroupSerializer
 
 
-class AlertGroupsView(ReadOnlyModelViewSet):
+class RetrieveViewSet(mixins.RetrieveModelMixin, viewsets.GenericViewSet):
+    """
+    A viewset that provides only `retrieve` actions.
+    """
+
+
+class AlertGroupsView(RetrieveViewSet):
```

Comment:
mixins.RetrieveModelMixin, viewsets.GenericViewSet can be arguments of AlertGroupsView directly

Reply:
True, I thought the intent was clearer this way 🤷 Happy either way though